.circleci/config.yml

```diff
@@ -92,7 +92,12 @@ aliases:
       command: |
         # Only test on external pull requests
         if [ -n "$CIRCLE_PR_NUMBER" ]; then
-          .ci/test.sh
+          cabal v2-test clash-prelude
+          cabal v2-test clash-lib
+          cabal v2-test clash-cores
+          cabal v2-test clash-cosim
+          cabal v2-run testsuite -- --hide-successes -p .VHDL
+          cabal v2-run testsuite -- --hide-successes -p .Verilog
```

Comment:
No love for `SystemVerilog`?

Reply:
No love :(. It needs modelsim.

Reply:
Not on circleCI: because non of the open source simulators support the subset of SV that Clash generates. Hopefully with the local gitlab runners we can test SV on gitlab.

Reply:
We can @christiaanb! At least one of the runners has modelsim installed now, so in theory we could activate it today. We've used for the closed project on GitLab already. I'd like to improve the way we manage the runners a bit before we commit to it though. Manually setting it all up is going to bite us.